journal/app/config/parameters.yml

```diff
@@ -3,5 +3,5 @@ parameters:
     secret: ThisTokenIsNotSoSecretChangeIt
     session_name: journal
     trusted_hosts:
-      - localhost
+      - '^.*$'
```

Comment:
what to limit it to?

Reply:
Probably can turn it off. It's a possible attack vector for misconfigured servers, which this isn't.